next/pages/bratislava.css

```diff
@@ -180,3 +180,60 @@ input[type='number']::-webkit-outer-spin-button {
 .form-group label.control-label {
   display: none;
 }
+
```

Comment:
add comment that this belongs to chatbot, or move all related css to separate file.